lib/get-config-for-markdown-file.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+var path = require('path');
+
+
+module.exports = function getConfigForMarkdownFile(inputFile) {
+  var configDir = path.dirname(inputFile);
+  var extName = path.extname(inputFile);
+  var configFile = path.basename(inputFile, extName) + '.json';
+
+  configFile = path.join(configDir, configFile);
+  console.error('checking for', configFile);
+  var configObj = {};
+  if (fs.existsSync(configFile))
+    configObj = JSON.parse(fs.readFileSync(configFile))
```

Comment:
Hmm... I think the process of creating html should abort and fail fast if the user feeds invalid json-configs to the process. 

The users intention is probably that the config should be used if they added it, but when adding it they made a typo - just continuing would result in a build with errors like wrong used templates .